xenit-alfresco/values.yaml

```diff
@@ -1,4 +1,5 @@
 general:
+  enterprise: false
```

Comment:
I would set the default to "true" otherwise an automatic upgrade (renovate bot) could break some production environments.

Reply:
Do not forget to update the changelog.md with changes and instructions on steps that are required after the upgrade.

Reply:
done